pkg/kube/watcher_test.go

```diff
@@ -130,13 +130,15 @@ func TestEventWatcher_EventAge_whenEventCreatedAfterStartupAndBeforeMaxAge(t *te
 
 	startup := time.Now().Add(-10 * time.Minute)
 	ew.setStartUpTime(startup)
-	// event is 8m after stratup time (2m in max age) -> expect processed
+	// event is 8m after startup time (2m in max age) -> expect processed
 	event1 := corev1.Event{
+		ObjectMeta: metav1.ObjectMeta{
+			CreationTimestamp: metav1.Time{Time: startup.Add(8 * time.Minute)},
+		},
```

Comment:
I'd create a local variable with the value of `startup.Add(8 * time.Minute)` and use it in both places to guarantees equality between the creation/last timestamps every time.